collection_requirements.rst

```diff
@@ -41,7 +41,9 @@ galaxy.yml
 ----------
 
 * tags MUST be set
-* dependencies MUST be set to ``'>=1.0.0'`` (Might have to think about dependency order when publishing)
+* dependencies MUST be set to ``'>=1.0.0'``
+  * This means that all dependencies have to specify lower bounds on the versions, and these lower bounds should be proper releases, and not versions of the form 0.x.y.
```

Comment:
I don't understand "proper releases, and not versions of the form 0.x.y.".

Reply:
I replaced `proper` by `stable`. Is it better this way?

Reply:
ah yes !